cli/src/commands/create/comments.rs

```diff
@@ -60,9 +63,17 @@ pub struct CreateCommentsArgs {
     #[structopt(short = "n", long = "no-charge")]
     /// Whether to attempt to bypass billing (internal only)
     no_charge: bool,
+
+    #[structopt(short = "w", long = "no-warning")]
+    /// Don't warn when users who have permission to bypass billing are not bypassing billing
+    suppress_no_charge_warning: Option<bool>,
```

Comment:
I'd say this flag should be either `-y` (yes) by convention with other linux tooling (see also: apt, kubectl, vogon etc.)